src/Layers/xrRenderGL/glR_Backend_Runtime.h

```diff
@@ -4,6 +4,8 @@
 
 #include "glStateUtils.h"
 
+extern CHW HW;
```

Comment:
My comment about this is in HW.h

In addition, this brokes all DirectX renderers, since you didn't included this change in `R_Backend_Runtime.h` and `dx10R_Backend_Runtime.h`.